### PR TITLE
Set date timezone

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: roles/php/tasks/main.yml [hunk 1]
```diff
@@ -9,3 +9,4 @@
  - include: activate_mongo_apache.yml
 # - include: activate_redis_cli.yml
  - include: activate_mongo_cli.yml
+ - include: php_cli_ini_set
```
Comment (alavieille) [inline roles/php/tasks/main.yml]: I think you forgot .yml

**Comment by nicolasThal** [comment]
:+1: with the .yml to add

**Comment by ngilain** [comment]
@itkg-alavieille : filename fixed

**Comment by alavieille** [comment]
:+1:

**Pull request merged**